pkg/disk/controllerserver.go

```diff
@@ -573,32 +561,26 @@ func (cs *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		if existsSnapshot.SourceDiskId == req.GetSourceVolumeId() {
 			csiSnapshot, err := formatCSISnapshot(&existsSnapshot)
 			if err != nil {
-				return nil, err
+				klog.Errorf("CreateSnapshot:: format snapshot failed: %v", err)
+				return nil, status.Errorf(codes.Internal, "format snapshot failed: %v", err)
```

Comment:
Since you have modified this, can you also remove all the logs immediately preceding return error? Interceptor will log the error automatically